scripts/nettraffic

```diff
@@ -48,7 +62,6 @@ do
       echo -e "  -e  (0|1|2|3|4)\t Level of units displayed. (0 for none, 1 for K, 2 for KB, 3 for K/s,"
```

Comment:
Not sure why, but the "-e" option is being printed to the console for me:
```
$ scripts/nettraffic -h
-e Usage: nettraffic [OPTION]
-e Options:
-e   -i  (%ifname%)      Specify which network interface to monitor. Required.
-e   -a  (up|dn|total)   Shows which data to display. Prints both if not used.
-e   -u  (K|KB|M|MB)     Units to show the data in. Automatic if not specified.
-e   -e  (0|1|2|3|4)     Level of units displayed. (0 for none, 1 for K, 2 for KB, 3 for K/s,
-e                       4 for KB/s). Default if 4.
-e   -l                  Choose whether or not to display the label (up or down).
-e   -t  (%integer%)     Set the interval for the data update. Default is 1.
-e   -v                  Prints version number.
-e   -h                  Prints this help.
For full help, visit http://github.com/ma-santoso/nettrafic/
```

I tried removing them from the echo commands and other than the tab characters ("\t") it's fine.  I suggest just removing the "-e" and tab characters.